dll/win32/imm32/ime.c

```diff
@@ -593,7 +593,11 @@ BOOL WINAPI ImmNotifyIME(HIMC hIMC, DWORD dwAction, DWORD dwIndex, DWORD_PTR dwV
     if (IS_NULL_UNEXPECTEDLY(pImeDpi))
         return FALSE;
 
+    TRACE("NotifyIME(%p, %lu, %lu, %p)\n", hIMC, dwAction, dwIndex, dwValue);
```

Comment:
Should we need to have this trace? (same question for the other one in utils.c)

Reply:
I think this is an important trace.